spec/OpenLineage.yml

```diff
@@ -0,0 +1,268 @@
+          type: string
+          format: date-time
+        nominalEndTime:
+          description: An [ISO-8601](https://en.wikipedia.org/wiki/ISO_8601) timestamp representing the nominal end time of the run.
+          type: string
+          format: date-time
+      required:
+        - nominalStartTime
+
+    ParentRunFacet:
+      description: the id of the parent run and job, iff this run was spawn from an other run (for example, the Dag run scheduling its tasks)
+      type: object
+      properties:
+        run:
+          type: object
+          properties:
+            runId:
+              type: string
+              format: uuid
+          required:
+            - runId
+        job:
+          properties:
+            namespace:
+              description: The namespace containing that job
+              type: string
+              example: "my-scheduler-namespace"
+            name:
+              description: The unique name for that job within that namespace
+              type: string
+              example: "myjob.mytask"
+          required:
+            - namespace
+            - name
+      required:
+        - run
+        - job
+
+## Job Facets
+    DocumentationJobFacet:
+      type: object
+      properties:
+        description:
+          description: The description of the job.
+          type: string
+      required:
+        - description
+
+    SourceCodeLocationJobFacet:
+      type: object
+      properties:
+        type:
+          type: string
+          example: git
+        url:
+          type: string
+          format: uri
+
+
+    SQLJobFacet:
+      type: object
+      properties:
+        query:
+          type: string
+          example: SELECT * FROM foo
+      required:
+        - query
+
+## Dataset facets
+    DocumentationDatasetFacet:
+      type: object
+      properties:
+        description:
+          description: The description of the dataset.
+          type: string
+          example: "canonical representation of entity Foo"
+      required:
+        - description
+
+    SchemaDatasetFacet:
+      type: object
+      properties:
+        fields:
+          description: The fields of the table.
+          type: array
+          items:
+            type: object
+            properties:
+              name:
+                description: The name of the field.
+                type: string
+                example: column1
+              type:
+                description: The type of the field.
+                type: string
+                example: VARCHAR|INT|...
+              description:
+                description: The description of the field.
+                type: string
+            required:
+              - name
+              - type
+
+    DatasourceDatasetFacet:
+      type: object
+      properties:
+        name:
+          type: string
+        url:
```

Comment:
Should the property also be called `uri`? I am assuming that a Datasource for a Dataset could be something like a Kafka topic; or at least something that wouldn't have a URL.

Reply:
that is correct. Thanks for the feedback. 
Feel free to reach out on the mailing list and slack or comment on the PRs. You can also open github issues for specific things you'd want to contribute.